crates/workspace/src/pane.rs

```diff
@@ -1133,11 +1133,19 @@ impl Pane {
             }
         }
 
-        // If a buffer is open both in a singleton editor and in a multibuffer, make sure
-        // to focus the singleton buffer when prompting to save that buffer, as opposed
-        // to focusing the multibuffer, because this gives the user a more clear idea
-        // of what content they would be saving.
-        items_to_close.sort_by_key(|item| !item.is_singleton(cx));
+        let active_item_id = self.active_item().unwrap().item_id();
```

Comment:
Is this `unwrap` here safe? What if we don't have an active item?

Reply:
Thank you for the review. I noticed that the logic here already checks whether items are empty, so I think this is safe.

However, I have changed it to use `Option::filter` to ensure that no unexpected behavior occurs.